be/CMakeLists.txt

```diff
@@ -217,6 +217,8 @@ message(STATUS "Using OpenSSL Root Dir: ${OPENSSL_ROOT_DIR}")
 add_library(crypto STATIC IMPORTED)
 set_target_properties(crypto PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libcrypto.a)
 
+add_library(AWS::crypto STATIC IMPORTED)
+set_target_properties(AWS::crypto PROPERTIES IMPORTED_LOCATION ${THIRDPARTY_DIR}/lib/libcrypto.a)
```

Comment:
shall use alias instead of import the crypto.a library again.

Reply:
Maybe you can take another pr, it's has been merged🤣